app/controllers/teams.js

```diff
@@ -18,8 +18,8 @@ export default class TeamsController extends Controller {
     return this.sortedTeamMembers.filter(inTeam('alumni'));
   }
 
-  get coreCLITeamMembers() {
-    return this.sortedTeamMembers.filter(inTeam('cli'));
+  get coreToolingTeamMembers() {
+    return this.sortedTeamMembers.filter(inTeam('tooling'));
   }
 
   get embroiderTeamMembers() {
```

Comment:
This getter can be removed now.

For some reason GitHub is not allowing me to put a multiline code suggestion here 🙃

Reply:
good catch, fixed 👍